tailwind.config.js

```diff
@@ -12,9 +12,9 @@ module.exports = {
     },
   },
   safelist: [
-      "bg-slate-200",
+      ...[...Array(100)].flatMap((item, index) => [`pl-[${index +1}px]`]),
```

Comment:
Have no idea what this does, but i trust you on this :laughing:

Reply:
This adds the tailwind classes "pl-1px" - "pl-100px" to the sefelist.